afm/filesystems/vault.py

```diff
@@ -34,34 +36,36 @@ def get_raw_secret_from_vault(jwt, secret_path, vault_address, vault_path, role)
         logger.error("Malformed vault authorization response")
         return None
     client_token = vault_auth_response["auth"]["client_token"]
-    logger.debug("client_token: %s", str(client_token))
     secret_full_path = vault_address + secret_path
-    logger.debug("secret_full_path = %s", str(secret_full_path))
     response = requests.get(secret_full_path, headers={"X-Vault-Token" : client_token})
-    logger.error("Status code from Vault response: " + str(response.status_code))
+    logger.debug('vault response',
+        extra={'secret_full_path': str(secret_full_path),
+               'status_code': str(response.status_code)})
```

Comment:
The key should be error, so that we match the keys that are generated in the manager.  Use a constant.  Please address in all error log entries.